app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -116,9 +116,17 @@ const rateLimitTimeouts = {};
 export default function createRPCMethodTrackingMiddleware({
   trackEvent,
   getMetricsState,
-  rateLimitSeconds = 60 * 5,
+  rateLimitSeconds,
   securityProviderRequest,
 }) {
+  let finalRateLimitSeconds = rateLimitSeconds;
+
+  ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+  finalRateLimitSeconds ||= 60 * 5;
+  ///: END:ONLY_INCLUDE_IN
+
+  finalRateLimitSeconds ||= 60;
+
```

Comment:
Instead of changing how this function's default parameter is specified, why not change it at the callsite?

Reply:
Also from what i'm reading the default for non-mmi was previously 300 seconds (60 * 5 * SECOND) and now in non-mmi it'll be 60 seconds and 300 seconds for MMI?

Reply:
Yes, it doesn't make sense, reverted back to original code